fe2-android/app/src/test/framework/robolectric/java/com/github/dedis/popstellar/di/TestKeysetModule.java

```diff
@@ -39,7 +40,7 @@ public static AndroidKeysetManager provideDeviceKeysetManager() {
       Ed25519PrivateKeyManager.registerPair(true);
       PublicKeySignWrapper.register();
       when(manager.getKeysetHandle())
-          .thenReturn(KeysetHandle.generateNew(Ed25519PrivateKeyManager.rawEd25519Template()));
+          .thenReturn(KeysetHandle.generateNew(KeyTemplates.get("ED25519_RAW")));
```

Comment:
Was that necessary ?

Reply:
`rawEd25519Template()` is currently deprecated (see [this](https://google.github.io/tink/javadoc/tink-android/HEAD-SNAPSHOT/deprecated-list.html))